dev-libs/libunistring/libunistring-0.9.9.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A library for manipulating Unicode strings"
+DESCRIPTION="libunistring provides functions for manipulating Unicode strings \
+and for manipulating C strings according to the Unicode standard."
+HOMEPAGE="https://www.gnu.org/software/libunistring/"
+COPYRIGHT="1998-2018 Free Software Fundation, Inc."
+LICENSE="GNU GPL v2
+	GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="https://ftpmirror.gnu.org/libunistring/libunistring-$portVersion.tar.gz
+	https://ftp.gnu.org/gnu/libunistring/libunistring-$portVersion.tar.gz"
+CHECKSUM_SHA256="f5e90c08f9e5427ca3a2c0c53f19aa38b25c500913510ad25afef86448bea84a"
+PATCHES="libunistring-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+libVersion="2.1.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+portVersionCompat="$portVersion compat >= ${portVersion%%.*}"
+
+PROVIDES="
+	libunistring$secondaryArchSuffix = $portVersionCompat
+	lib:libunistring$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libiconv$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libunistring${secondaryArchSuffix}_devel = $portVersion
+	devel:libunistring$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	libunistring$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libiconv$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:autoconf
```

Comment:
```diff
-	cmd:autoconf
+	cmd:autoreconf

Reply:
Good catch, thanks @alaviss!
Hopefully this does not (currently) have any impact.

Regarding the patchset, you are also right. We can drop it. (I have checked this on each arch, including x86 second arch.
The resulting shared lib is identical.
For the static lib, the checksums do not match but this is due to the fact that, unlike Debian, we haven't implemented reproductible builds yet.